test/includes/clustering.sh

```diff
@@ -82,6 +82,7 @@ ip link set dev "${veth2}" name eth0
 ip link set eth0 up
 ip addr add "10.1.1.10${id}/16" dev eth0
 ip route add default via 10.1.1.1
+ip link add localBridge${id} type bridge
```

Comment:
can we access the net ns via id to only add the per-member dummy nics in the test_clustering_network

Also this is removing the interface not needed because its destroyed when the netns is destroyed?

Reply:
Yes, destroying should not be needed as netns gets destroyed after the test run.

I tried to move the dummy bridge setup into the test code, but it turned out quite verbose. Because we need to talk to nsenter, and getting the right path to read the `nspid` is not straight forward:
```
id="${1}"
shift
prefix="lxd$$"
ns="${prefix}${id}"
```
Especially the 2nd and 3rd line are a bit of a mystery to me. Duplicating those lines in the test seemed like a bad idea, as it clashes with the single source of truth. That's why I resorted to adding a local bridge on each node with the id of that node in the setup code directly, where the ns pid is already resolved and in use.